configs/simplifions/config.yaml

```diff
@@ -232,6 +236,35 @@ pages:
             name: ⭐️⭐️ Dites-le nous une fois | L'usager n'a plus à fournir de justificatifs
           - id: acces-facile
             name: ⭐️ Accès facile | L'usager ou l'agent trouve l'information
+
```

Comment:
Out of PR scope

Reply:
Oh, right ! Sorry. I didn't go see the file as you were just pointing to one line.